client/checkout/upe-styles/index.js

```diff
@@ -234,27 +257,23 @@ const hiddenElementsForUPE = {
 	},
 };
 
-export const getFieldStyles = ( selector, upeElement, focus = false ) => {
+export const getFieldStyles = ( selector, upeElement ) => {
 	if ( ! document.querySelector( selector ) ) {
 		return {};
 	}
 
 	const validProperties = upeRestrictedProperties[ upeElement ];
 
 	const elem = document.querySelector( selector );
-	if ( focus ) {
-		elem.focus();
-	}
 
 	const styles = window.getComputedStyle( elem );
 
 	const filteredStyles = {};
-
 	for ( let i = 0; i < styles.length; i++ ) {
 		const camelCase = dashedToCamelCase( styles[ i ] );
 		if ( validProperties.includes( camelCase ) ) {
-			filteredStyles[ camelCase ] = styles.getPropertyValue(
-				styles[ i ]
+			filteredStyles[ camelCase ] = maybeConvertRGBAtoRGB(
+				styles.getPropertyValue( styles[ i ] )
```

Comment:
AFAICT, Stripe only allows `rgba` colours with transparency for border outlines; otherwise the appearances API will ignore them and present a warning in the console.